src/SoundList.js

```diff
@@ -12,6 +12,7 @@ export function formatTime(secs) {
   // Pads a zero in front of a number if it's less than 10
   const padLeft = (num) => `${num < 10 ? "0" : ""}${num}`;
   // e.g., 1 => 01, 9 => 09, 10 => 10, 59 => 59
+  if (hrs === 0) return `${padLeft(mins)}:${padLeft(remainingSecs)}`;
```

Comment:
nit: we could ternary here

Reply:
It actually uses one more line of code to use the ternary operator, and makes it potentially less human readable, at least for new JavaScript developers, but we can still use a ternary if you prefer.

Original:
```
  if (hrs === 0) return `${padLeft(mins)}:${padLeft(remainingSecs)}`;
  return `${padLeft(hrs)}:${padLeft(mins)}:${padLeft(remainingSecs)}`;
```

Ternary after automatic formatting on save:
```
  return hrs === 0
    ? `${padLeft(mins)}:${padLeft(remainingSecs)}`
    : `${padLeft(hrs)}:${padLeft(mins)}:${padLeft(remainingSecs)}`;
```

Reply:
Latter approach is considered more idiomatic in js than the first because it slightly reduces cognitive overhead

Reply:
One of the key benefits of ternaries is that it makes it very easy to visually see the differences between the different return values because they're side by side. One of the things I can't stress enough is that counting LOC isn't actually a good way of determining if something is simpler (but this isn't intuitive at first, takes time to see what I mean).

Reply:
I agree that the ternary makes it easier to visually see the difference in the two potential return Strings.

The commit I just made uses a ternary.